mmhuman3d/utils/geometry.py

```diff
@@ -60,7 +60,7 @@ def rot6d_to_rotmat(x):
     Output:
         (B,3,3) Batch of corresponding rotation matrices
     """
-    x = x.view(-1, 3, 2)
+    x = x.reshape(-1, 3, 2)
```

Comment:
It seems we can refactor geoemtry to accept both numpy and tensor?

Reply:
Add an 'if' statement to accept both numpy and tensor.